catalog/ftp/src/main/java/ddf/catalog/ftp/FtpServerManager.java

```diff
@@ -252,7 +252,8 @@ private void waitForConnections() {
           break;
         }
       } catch (InterruptedException e) {
-        Thread.interrupted();
+        Thread.interrupted(); // <<<<<<<< Do we still need this line?
```

Comment:
❗️ This line should be removed as it is designed to check the interrupted flag which you know would always return true since you got an interrupted exception.